### DISCUSSION TITLE
Output prediction mask is empty

### DISCUSSION THREAD

**Discussion by junxiant** [Project-MONAI/MONAILabel#1316] [Q&A] [answered]
hi everyone,

I'm trying to do a 2D segmentation task by using my own pretrained model which i have
 trained using pytorch. I have followed the instructions from [https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#how-to-add-new-model](url). My .pt model is under /model/pretrained_{name}.pt . 

Everything looks fine and runs without error, 
but the output prediction masks are empty.

For example, this is the output using MonaiLabel's:
![cv3](https://user-images.githubusercontent.com/25401067/219332334-e1e8936b-19a8-4e77-bf4e-e194cfcedda2.png)

This is the output when i ran my own test.py using pytorch:
![pytorch_script](https://user-images.githubusercontent.com/25401067/219332288-f0c253d9-5d64-4844-bab0-fb579a1a33a5.png)

Did i miss out something when trying to implement my own transforms?

**Comment by junxiant** [comment, marked as answer]
Solved with an issue with Normalization in pre_transforms
